common/djangoapps/student/tests/tests.py

```diff
@@ -404,6 +414,91 @@ def test_refundable_when_certificate_exists(self):
         self.assertFalse(enrollment.refundable())
 
     @unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
+    def test_refundable_when_refund_window(self):
+        CourseModeFactory.create(
```

Comment:
This is duplicated three times. Move it to `setUp`. Also, I just noticed that this file is HUGE! At a minimum, break the refund tests into their own file.